backend/settings.py

```diff
@@ -220,6 +220,9 @@
         "toolbar": "formatselect | bold italic backcolor | "
             "bullist numlist outdent indent | removeformat | code | help",
         "default_link_target": "_blank",
+        "invalid_elements": "span, h1, h2, h3, h4, h5, h6",
+        "invalid_styles": "color, font-family, background-color"
```

Comment:
This seems to be working pretty well. I think we could allow `span`, there might be a use for it.

I'd probably want to check this with @zqian and Jeff next week to make sure they're on board with the change.

Reply:
Yeah, I'd also like to get more examples on what is really worth removing, most things are defined as classes rather than inline styles. 

It's nice that it's configurable here though. Most things on the web are probably defined as a div. 

I think if we removed these we'd also want to remove these options from the "Format" menu dropdown too, which would probably require [fully configuring the menu. ](https://www.tiny.cloud/docs/configure/editor-appearance/#menu)

Reply:
@roujiawang, can you look into restricting the available element types from the menus, specifically headers?

Reply:
If we can come up with a few test cases as well, that can be stored somewhere, that would be great.

Reply:
We may need to restrict the class attribute or only allow some classes as well. We'd have to look at the test cases and see what they include.

Reply:
The settings for text box toolbar menu can be changed by overriding certain default features inside the braces of TINYMCE_DEFAULT_CONFIG in settings.py. For instance, adding _"block_formats": "",_ would clear out font size options in the "Paragraph" section of the toolbar menu, making admin editors unable to set font size h1~h6.
I could launch a new PR about this.

Reply:
There may be gentler ways to disable certain html elements, like the "Using remove" example in the middle of https://www.tiny.cloud/docs/configure/content-formatting/

Reply:
I think the idea was we'd like you to disable these options as part of this PR. I think the first approach sounds more promising. "using remove" doesn't seem to actually disable the elements, just modify the behavior of the "Remove formatting" button.